caffe2/python/operator_test/conv_transpose_test.py

```diff
@@ -37,6 +38,11 @@ def test_convolution_transpose_layout_legacy_args(
         b = np.random.rand(output_channels).astype(np.float32) - 0.5
         outputs = {}
         for order in ["NCHW", "NHWC"]:
+            # NHWC not supported in MIOpen, run HIP
+            if hiputl.run_in_hip(gc, dc) and engine == "CUDNN":
```

Comment:
Ok I think you can directly override engine, so instead of an if-else, you can do

```
if hiputil.run_in_hip(gc, dc) and order == 'NHWC':
  engine = ''
```

Reply:
Thank you!!